CSharp/DevVmPowershell/Helpers/Implementations/RestHelper.cs

```diff
@@ -27,25 +27,25 @@ public static HttpClient GetHttpClient(string instanceAddress, string adminUsern
 			return httpClient;
 		}
 
-		public static HttpResponseMessage MakePost(HttpClient httpClient, string url, string request)
+		public static async Task<HttpResponseMessage> MakePostAsync(HttpClient httpClient, string url, string request)
```

Comment:
We should not have these methods as static. We are not adding unit tests now, but if in future we want to add unit tests, having static methods will not make it possible.

- [ ] Can we convert them to not static methods and do DI where ever RestHelper is needed.